elements/pfe-button/demo/pfe-button.html

```diff
@@ -5,175 +5,90 @@
-  <pfe-button size="small" variant="tertiary"><button>Tertiary small</button></pfe-button>
-  <pfe-button size="small" danger><button>Danger small</button></pfe-button>
-  <pfe-button size="small" warning><button>Warning small</button></pfe-button>
-
-  <pfe-button size="large"><button>Primary large</button></pfe-button>
-  <pfe-button size="large" variant="secondary"><button>Secondary large</button></pfe-button>
-  <pfe-button size="large" variant="secondary" danger><button>Danger secondary large</button></pfe-button>
-  <pfe-button size="large" variant="tertiary"><button>Tertiary large</button></pfe-button>
-  <pfe-button size="large" danger><button>Danger large</button></pfe-button>
-  <pfe-button size="large" warning><button>Warning large</button></pfe-button>
+  <pfe-button size="small">Primary small</pfe-button>
+  <pfe-button size="small" variant="secondary">Secondary small</pfe-button>
+  <pfe-button size="small" variant="secondary" danger>Danger secondary small</pfe-button>
+  <pfe-button size="small" variant="tertiary">Tertiary small</pfe-button>
+  <pfe-button size="small" danger>Danger small</pfe-button>
+  <pfe-button size="small" warning>Warning small</pfe-button>
+
+  <pfe-button size="large">Primary large</pfe-button>
+  <pfe-button size="large" variant="secondary">Secondary large</pfe-button>
+  <pfe-button size="large" variant="secondary" danger>Danger secondary large</pfe-button>
+  <pfe-button size="large" variant="tertiary">Tertiary large</pfe-button>
+  <pfe-button size="large" danger>Danger large</pfe-button>
+  <pfe-button size="large" warning>Warning large</pfe-button>
 </section>
 
 <section>
   <h2>Progress</h2>
   <p>Stateful buttons have a <code>loading</code> indicator</p>
-  <pfe-button id="loading-logs">
-    <button>Resume loading logs</button>
-  </pfe-button>
+  <pfe-button id="loading-logs">Resume loading logs</pfe-button>
   <!-- Custom ARIA-label for progress indicator -->
-  <pfe-button id="loading-scdr" variant="secondary" progress-label="Custom progress label">
-    <button>Click to start loading</button>
-  </pfe-button>
+  <pfe-button id="loading-scdr" variant="secondary" progress-label="Custom progress label">Click to start loading</pfe-button>
   <pfe-button id="loading-icon" plain>
-    <button>
-      <svg fill="currentColor" height="1em" width="1em" viewBox="0 0 512 512" aria-label="upload">
-        <path d="M296 384h-80c-13.3 0-24-10.7-24-24V192h-87.7c-17.8 0-26.7-21.5-14.1-34.1L242.3 5.7c7.5-7.5 19.8-7.5 27.3 0l152.2 152.2c12.6 12.6 3.7 34.1-14.1 34.1H320v168c0 13.3-10.7 24-24 24zm216-8v112c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V376c0-13.3 10.7-24 24-24h136v8c0 30.9 25.1 56 56 56h80c30.9 0 56-25.1 56-56v-8h136c13.3 0 24 10.7 24 24zm-124 88c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20zm64 0c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20z"></path>
-      </svg>
-    </button>
+    <svg fill="currentColor" height="1em" width="1em" viewBox="0 0 512 512" aria-label="upload">
```

Comment:
Use offscreen text and aria-hidden on SVG. See https://www.sarasoueidan.com/blog/accessible-icon-buttons/